src/main/java/org/jvnet/hudson/test/RealJenkinsRule.java

```diff
@@ -759,6 +758,15 @@ public String[] getTruststoreJavaOptions() {
      * If you need to pass and/or return values, you can still use a static method reference:
      * try {@link #runRemotely(Step2)} or {@link #runRemotely(StepWithReturnAndOneArg, Serializable)} etc.
      * (using {@link XStreamSerializable} as needed).
+     * <p>
+     * Alternately, you could use a lambda (taking care not to capture non-serializable objects from scope):
```

Comment:
In my experience, this is pretty difficult to assess once you experience an issue.

Reply:
Could be. Noted in https://github.com/jenkinsci/jenkins-test-harness/pull/881#discussion_r1866482652. The difference is that (anonymous) inner classes will _always_ capture variables in lexical scope, whereas lambdas usually will not. Noted another caveat in b9e40f53ae5faca982a045f7b23a7825ed228ffb.